### ISSUE TITLE
Test latest translations

### ISSUE THREAD

**Issue by scootergrisen** [schuellerf/xfce-test#11]
I want to test Xfce with the latest translation from Transifex so i can test the strings and improve the translations.

Can you write something that can help me do this like how to change language, how to get the translation in, how to update the translations, etc.

**Comment by schuellerf** [comment]
...probably ...
I'm not too familiar with transifex - is there a set of commands to get the latest transifex stuff or just updating the source? If you already have a set of commands for that it would help...

**Comment by scootergrisen** [comment]
If use the following when i want to download the translations:

    tx init --skipsetup

    tx set --auto-remote https://www.transifex.com/xfce/exo/
    tx set --auto-remote https://www.transifex.com/xfce/garcon/
    tx set --auto-remote https://www.transifex.com/xfce/libxfce4ui/
    tx set --auto-remote https://www.transifex.com/xfce/libxfce4util/
    tx set --auto-remote https://www.transifex.com/xfce/thunar/
    tx set --auto-remote https://www.transifex.com/xfce/thunar-plugins/
    tx set --auto-remote https://www.transifex.com/xfce/thunar-volman/
    tx set --auto-remote https://www.transifex.com/xfce/tumbler/
    tx set --auto-remote https://www.transifex.com/xfce/xfce-apps/
    tx set --auto-remote https://www.transifex.com/xfce/xfce-panel-plugins/
    tx set --auto-remote https://www.transifex.com/xfce/xfce-www/
    tx set --auto-remote https://www.transifex.com/xfce/xfce4-appfinder/
    tx set --auto-remote https://www.transifex.com/xfce/xfce4-panel/
    tx set --auto-remote https://www.transifex.com/xfce/xfce4-power-manager/
    tx set --auto-remote https://www.transifex.com/xfce/xfce4-session/
    tx set --auto-remote https://www.transifex.com/xfce/xfce4-settings/
    tx set --auto-remote https://www.transifex.com/xfce/xfconf/
    tx set --auto-remote https://www.transifex.com/xfce/xfdesktop/
    tx set --auto-remote https://www.transifex.com/xfce/xfwm4/

    tx pull -l da

I have no idea if this is usefull for xfce-test but at least i can get the files without having to use a browser.
But its slow.

https://docs.transifex.com/client/introduction

**Comment by schuellerf** [comment]
Do I need to join the xfce-team in transifex somehow to be able to pull the translations?

**Comment by scootergrisen** [comment]
I'm not sure.
It would seem so. At least the way i do it.
I get "tx ERROR: Forbidden" when i try to pull translations from a project i am no longer a member off.

**Comment by Skunnyk** [comment]
Yes you need an account on transifex to be able to tx pull the resources.

But if you build yourself the xfce-test docker image, it will compile the latest git master of a all "visual" components (panel, thunar, settings, session, desktop, xfwm etc), so you can see if all translation are correct in their context before a release :)

**Comment by schuellerf** [comment]
Thanks @Skunnyk for the hint! I also came to this conclusion and will keep this issue until
"make language-test" is more user-friendly.
@scootergrisen you can try to get "make language-test" running in the meantime...
This will create a folder called "lang-screenshots" with all languages of xfce4-display-settings just as a test for now...
